<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgBoxDlg</class>
 <widget class="QDialog" name="MsgBoxDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>184</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>394</width>
    <height>184</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>394</width>
    <height>184</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="m_wgtBg">
     <property name="minimumSize">
      <size>
       <width>394</width>
       <height>184</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>394</width>
       <height>184</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#m_wgtBg
{
border-radius:10px 10px 0px 0px;
border: 1px solid #8D8D8D;
background-color:#FFFFFF;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_1" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>394</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>394</width>
          <height>38</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>394</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>394</width>
          <height>39</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>28</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>28</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="m_lbQuestion">
           <property name="minimumSize">
            <size>
             <width>154</width>
             <height>19</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>336</width>
             <height>19</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-family:'Malgun Gothic','맑은고딕','돋움',Dotum,sans-serif;
color:#000000;
font-size:14px;
font-weight:bold;
letter-spacing:-1px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>394</width>
          <height>56</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>394</width>
          <height>56</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>28</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>28</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="m_lbQuestion2">
           <property name="minimumSize">
            <size>
             <width>336</width>
             <height>56</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>336</width>
             <height>56</height>
            </size>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">font-family:'Malgun Gothic','맑은고딕','돋움',Dotum,sans-serif;
color:#000000;
font-size:14px;
font-weight:normal;
letter-spacing:-1px;</string>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="minimumSize">
         <size>
          <width>394</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>394</width>
          <height>40</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="leftMargin">
          <number>28</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>28</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnYes">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>94</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>94</width>
             <height>30</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::ActionsContextMenu</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">#m_btnSubmit {
	background-color: #00A3FF;
	border-radius:15px 15px 0px 0px;
 	transition-duration:0.5s;

	font-family:'Malgun Gothic','맑은고딕','돋움',Dotum,sans-serif;
	color:#F9F9F9;
	font-size:12px;
	font-weight:bold;
	letter-spacing:-1px;
}

#m_btnSubmit:hover {
	background-color: #0C61B6;
	box-shadow: 0px 14px 24px rgba(85, 193, 239, 0.25);
}

#m_btnSubmit:pressed {
	background-color: #054B92;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnNo">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>94</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>94</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#m_btnCancel {
	background-color: rgba(0, 0, 0, 0);
	border-radius:15px 15px 0px 0px;
	border: 1px solid #DCE5E5;

	font-family:'Malgun Gothic','맑은고딕','돋움',Dotum,sans-serif;
	color: #323232;
	font-size: 12px;
	font-weight: bold;
	letter-spacing: -1px;
}

#m_btnCancel:hover {
	border: 1px solid #C4D6D6;
}

#m_btnCancel:pressed {
	border: 1px solid #A3B5B5;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="minimumSize">
         <size>
          <width>394</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>394</width>
          <height>21</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>28</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>28</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
